lib/train/transports/docker.rb

```diff
@@ -83,6 +83,17 @@ def uri
       end
     end
 
+    def unique_identifier
+      uuid = @id # default uuid set to the docker host.
```

Comment:
This should default to `@container.id`, not `@id` - otherwise on windows we will have the same ID for all containers, that of the host.

Reply:
I am not sure why it will have the same id for all the containers on Windows. But yes I will add make the change to use @conainer.id if not @container.nil?